src/app/research/research.component.html

```diff
@@ -1,42 +1,40 @@
 <h1 class="center">Full Publication List</h1>
 
-<div>
-  <ul class="center publication-font" style="list-style: none;">
-    <li>
-      <b><u>LIBSA - A method for the determination of ligand-binding preference to allosteric sites on receptor ensembles</u></b>
+<div class="center publication-font">
+    <div>
+      <b><u><a href="http://pubs.acs.org/doi/pdf/10.1021/ci400474u">LIBSA - A method for the determination of ligand-binding preference to allosteric sites on receptor ensembles</a></u></b>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <b> tag by <strong>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)